frameworks/kafka/tests/client.py

```diff
@@ -93,7 +93,7 @@ def install(self, kerberos: sdk_auth.KerberosEnvironment=None) -> dict:
             "container": {
                 "type": "MESOS",
                 "docker": {
-                    "image": "elezar/kafka-client:4b9c060",
+                    "image": "elezar/kafka-client:deca3d0",
```

Comment:
Ya'll, I hate to be that guy but...

1. We need to move this dockerfile into the kafka directory of this repo
2. We need to host the docker image in mesopshere docker somewhere (ideally with a jenkins job to update it)

Reply:
Yes, there is a ticket for it.

Reply:
This one? https://jira.mesosphere.com/browse/DCOS-29527

Reply:
This seems like the perfect PR to fix this then :)

Reply:
I would prefer to address this in a separate PR.

Reply:
(The reason for doing this in a separate PR is that the build tooling would also need to be updated, so there are a number of moving parts)